test/built-ins/Temporal/PlainYearMonth/prototype/since/roundingincrement-non-integer.js

```diff
@@ -20,5 +20,6 @@ const earlier = new Temporal.PlainYearMonth(2000, 5);
 const later = new Temporal.PlainYearMonth(2000, 10);
 const result = later.since(earlier, { roundingIncrement: 2.5, roundingMode: "trunc" });
 TemporalHelpers.assertDuration(result, 0, 4, 0, 0, 0, 0, 0, 0, 0, 0, "roundingIncrement 2.5 truncates to 2");
-const result2 = later.since(earlier, { largestUnit: "months", smallestUnit: "months", roundingIncrement: 1e9 + 0.5, roundingMode: "expand" });
-TemporalHelpers.assertDuration(result2, 0, 1e9, 0, 0, 0, 0, 0, 0, 0, 0, "roundingIncrement 1e9 + 0.5 truncates to 1e9");
+// Cannot test the upper bound of 1e9 + 0.5 here, because the duration is
```

Comment:
This is also in a PR a bit early, right?

Reply:
I'm pretty sure this is no longer possible to test in this specific case because of removing the loop in RoundDuration. It would now overflow the calendar calculation because 1970-01-01 ± 10⁹ months is outside the range of representable calendar dates. (So maybe that is an example of a "use case" that the looping algorithm enabled, that isn't supported by the simpler calculation we have now.)

Note that this "upper bound + 0.5" case is only impossible to test for PlainYearMonth.p.since/until. Rounding to smaller units than months is not allowed because it doesn't make any sense for PlainYearMonth. For other types, we use a unit such as days that isn't going to need calendar arithmetic to balance up, e.g. [here](https://github.com/tc39/test262/blob/main/test/built-ins/Temporal/PlainDate/prototype/until/roundingincrement-non-integer.js#L23).